libraries/AC_Avoidance/AP_OADatabase.cpp

```diff
@@ -418,12 +373,15 @@ void AP_OADatabase::send_adsb_vehicle(mavlink_channel_t chan, uint16_t interval_
             continue;
         }
 
+        // convert object's position as an offset from EKF origin to Location
+        const Location item_loc(Vector3f(_database.items[idx].pos.x * 100.0f, _database.items[idx].pos.y * 100.0f, 0));
+
         mavlink_msg_adsb_vehicle_send(chan,
             idx,
-            _database.items[idx].loc.lat,
-            _database.items[idx].loc.lng,
+            item_loc.lat,
+            item_loc.lng,
             0,                          // altitude_type
-            _database.items[idx].loc.alt,
+            0,                          // altitude is always zero
```

Comment:
should be our alt above sea level?

Reply:
Thanks, updated so it sends the adsb messages using the vehicle's current altitude.